src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -153,7 +153,7 @@ public AbstractEntityAIGuard(@NotNull final J job)
         super.registerTargets(
           new AITarget(DECIDE, this::decide, GUARD_TASK_INTERVAL),
           new AITarget(GUARD_PATROL, this::shouldSleep, () -> GUARD_SLEEP, SHOULD_SLEEP_INTERVAL),
-          new AITarget(GUARD_PATROL, this::decide, GUARD_TASK_INTERVAL),
+          new AITarget(GUARD_PATROL, this::decide, 100),
           new AITarget(GUARD_SLEEP, this::sleep, 1),
```

Comment:
shouldn't we set the Guard_task_interval 100 instead?

Reply:
ye think that should be fine for all tasks

Reply:
oh not sure, didn't know the others had that interval as well. Because if we had 100, detection of enemies will be way too slow.